_overviews/contributors/index.md

```diff
@@ -0,0 +1,809 @@
+For instance, if your repository contains a `build.sbt` file in the root directory, Travis CI executes the
+`sbt ++$TRAVIS_SCALA_VERSION test` command, where the `TRAVIS_SCALA_VERSION` variable is, by default, set to an
+arbitrary Scala version (`2.12.8`, at the time these lines are written), which could be inconsistent with the
+`scalaVersion` defined in your `build.sbt` file.
+
+To avoid this potential inconsistency, you want to use one Scala version definition as a single source of truth.
+For instance, the [sbt-travisci](https://github.com/dwijnand/sbt-travisci) plugin lets you define the Scala version
+in the `.travis.yml` file, and then forwards this version to your sbt build definition. Alternatively, you can
+override the default command run by Travis CI to use the Scala version defined by the `scalaVersion` settings of
+your build.
+
+The latter approach is the one used in this guide. Override the command run by Travis CI by adding the folliwng
+lines to your `.travis.yml` file:
+
+~~~ yaml
+jobs:
+  include:
+    - stage: test
+      script: sbt test
+~~~
+
+Travis CI will now execute the `sbt test` command, which uses the Scala version from the build definition.
+
+Last, an important thing to setup is caching, to avoid the CI server to re-download your project dependencies each
+time it runs. For instance, in case you use sbt, you can instruct Travis CI to save the content of the `~/.ivy2/`
+and `~/.sbt/` directories across builds by adding the following lines to your `.travis.yml` file:
+
+~~~ yaml
+# These directories are cached at the end of the build
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt
+before_cache:
+  # Cleanup the cached directories to avoid unnecessary cache updates
+  - rm -fv $HOME/.ivy2/.sbt.ivy.lock
+  - find $HOME/.ivy2/cache -name "ivydata-*.properties" -print -delete
+  - find $HOME/.sbt        -name "*.lock"               -print -delete
+~~~
+
+For reference, here is our complete
+[.travis.yml example file](https://github.com/scalacenter/library-example/blob/master/.travis.yml).
+
+## Publish a Release
+
+Most build tools resolve third-party dependencies by looking them up on public repositories such as
+[Maven Central](https://search.maven.org/) or [Bintray](https://bintray.com/). These repositories host
+the library binaries as well as additional information such as the library authors, the open source
+license, and the dependencies of the library itself. Each release of a library is identified by
+a `groupId`, an `artifactId`, and a `version` number. For instance, consider the following dependency
+(written in sbt’s syntax):
+
+~~~ scala
+"org.slf4j" % "slf4j-simple" % "1.7.25"
+~~~
+
+Its `groupId` is `org.slf4j`, its `artifactId` is `slf4j-simple`, and its `version` is `1.7.25`.
+
+To make your library usable by other people, we recommend publishing it to the Maven Central repository.
```

Comment:
Maybe insert a sentence, at most two, about why we're recommending that over Bintray/JCenter.

Reply:
I’m not sure exactly why, actually. I remember people telling me that bintray was quite unstable. Also, Maven Central releases are immutable.

Am I correct? Do I miss something?

Reply:
I don't know. If neither of us has a reason, perhaps dodge the issue by changing from "We recommend X" to "In this example, we use X".